components/eam/docs/tech-guide/cosp.md

```diff
@@ -0,0 +1,36 @@
+# Cloud Feedback Model Intercomparison Project (CFMIP) Observation Simulator Package
+
+## Overview
+
+The Cloud Feedback Model Intercomparison Project (CFMIP) Observation Simulator Package (COSP; Bodas-Salcedo et al., 2011; Swales et al., 2018) was developed to improve the consistency between model clouds and satellite observations. COSP contains several independent satellite simulators for better comparing model clouds with satellite measurements collected by the International Satellite Cloud Climatology Project (ISCCP), the Moderate Resolution Imaging Spectroradiometer (MODIS), the Multi-angle Imaging SpectroRadiometer (MISR), Cloud-Aerosol Lidar and Infrared Pathfinder Satellite Observation (CALIPSO), and CloudSat. The use of satellite simulators will not only make a fairer comparison between model clouds and satellite data but also allow a more in-depth analysis of clouds. For example, clouds can be assessed in terms of their optical properties and vertical location, which dictate their radiative effects.
+
+### References
+
+- Zhang et al. (2024) [@zhang_understanding_2024]
```

Comment:
Don't break out the references separately. Keep the footnote references inline as done in other sections like P3 and MAM.